harbors/tests/test_harbors_graphql.py

```diff
@@ -0,0 +1,79 @@
+import pytest
+from graphene.test import Client
+
+from berth_reservations.schema import schema
+from harbors.models import BoatType, Harbor
+
+
+@pytest.fixture
```

Comment:
why are you repeating the fixtues, that are already defined in `conftest.py`?

Reply:
you should be able to do `def test_get_boat_type(boat_type):` without repeating these :)